buddyboss/buddyboss-platform
# Timeline preview image is not working

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2345 .

### How to test the changes in this Pull Request:

1. Go to this client site https://personaldevelopmentexpert.com/
2. Navigate to timeline and upload PDF file.
3. See error, PDF preview image will show blank.

### Proof Screenshots or Video

https://www.loom.com/share/5bdd69a942c3434b948612d415685e25

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
